### DISCUSSION TITLE
How to use ktx decode

### DISCUSSION THREAD

**Discussion by Pikn1K** [KhronosGroup/KTX-Software#908] [Q&A]
How to use ktx decode I downloaded the windows version in the bin exe files folder but nothing happens when opening

**Comment by MarkCallow** [comment]
We don't have a `ktx decode` command. What command are you actually talking about and what do  you expect it to do?

**Comment by MarkCallow** [comment]
Requested information not provided after almost one year. No further discussion possible.
